frontend/src/API/TeamEventsAPI.ts

```diff
@@ -42,7 +42,8 @@ export class TeamEventsAPI {
         });
         return [];
       }
-      return val.allTeamEventInfo as TeamEventInfo[];
+      const events = val.allTeamEventInfo as TeamEventInfo[];
+      return events;
```

Comment:
It's unnecessary to break this up into 2 lines.

Reply:
My bad, was testing some stuff.